src/components/Layout/Cluster/Cluster.tsx

```diff
@@ -40,28 +51,34 @@ const Wrapper = styled.div`
   }
 `
 const Body = styled.div<{
+  rowGap?: CharRelativeSize | AbstractSize
+  columnGap?: CharRelativeSize | AbstractSize
   gap: CharRelativeSize | AbstractSize
   align?: alignMethod
   justify?: justifyMethod
-}>(
-  ({ gap, align, justify }) => css`
+}>(({ rowGap, columnGap, gap, align, justify }) => {
+  const verticalGap = rowGap != null ? useSpacing(rowGap) : useSpacing(gap)
+  const horizontalGap = columnGap != null ? useSpacing(columnGap) : useSpacing(gap)
```

Comment:
null より undefined きそう、0 はこないので、こうかしら。

```suggestion
  const verticalGap = rowGap ? useSpacing(rowGap) : useSpacing(gap)
  const horizontalGap = columnGap ? useSpacing(columnGap) : useSpacing(gap)
```

Reply:
`!=` は `null` か `undefined` じゃなければ、って意図でした。わかりにくいかなぁ。 `!== undefined` のほうがいいかな。

さらに `0` は `gap` の値として入れられるようにしたいお気持ち。

Reply:
> `0` は `gap` の値として入れられるようにしたい

わかる。Stack と LineUp にも影響あるから別 PR ですかねぇ。

> !== undefined

この方がいいと思います!

Reply:
> `gap` で一括指定できると嬉しいかも

string をパースはやだなぁ。型の恩恵も受けにくい気がするし。オブジェクトでも受け付けるとかはありかも？

```TypeScript
gap: CharRelativeSize | AbstractSize | {
  row: CharRelativeSize | AbstractSize
  column: CharRelativeSize | AbstractSize
}
```

Reply:
ありかも!

Reply:
#1841 gap 0 で受け取れるやつ PR 出しといた